packages/core/src/modules/company/company-event-service.ts

```diff
@@ -0,0 +1,26 @@
+import { Company, Event } from "@dotkomonline/types"
+import { Cursor } from "../../utils/db-utils"
+import { CompanyEventRepository } from "./company-event-repository" // Note: you might need to create or rename this file based on previous changes.
+
+export interface CompanyEventService {
+  createEvent: (company: Company["id"], event: Event["id"]) => Promise<void>
+  deleteEvent: (company: Company["id"], event: Event["id"]) => Promise<void>
+  getEventsByCompanyId: (company: Company["id"], take: number, cursor?: Cursor) => Promise<Event[]>
+}
+
+export class CompanyEventServiceImpl implements CompanyEventService {
+  constructor(private readonly companyEventRepository: CompanyEventRepository) {}
+
+  async createEvent(company: Company["id"], event: Event["id"]) {
```

Comment:
Still needs return types

Reply:
I'm not sure about the interface pattern we want for many-to-many inserts in repositories

Reply:
For example, in 'event-company-repository.ts'

  createCompany: (id: Event["id"], company: Company["id"]) => Promise<void>
  async createCompany(id: Event["id"], company: Company["id"]) {
    await this.db
      .insertInto("eventCompany")
      .values({
        eventId: id,
        companyId: company,
      })
      .returningAll()
      .executeTakeFirst()
  }
  
  Is it an oversight that we are running 
  `.returningAll()
      .executeTakeFirst()
      ` here when we are not returning it?

Reply:
I ended up just removing create/update/delete for the company-event relation here as it is not used by the company section on the dashboard.